android/app/src/main/java/com/google/codeu/chatme/view/adapter/UserListAdapter.java

```diff
@@ -97,5 +111,24 @@ private void setHolderPicture(String picUrl) {
                         .into(this.civUserPic);
             }
         }
+
+        @Override
+        public void onClick(View view) {
+
```

Comment:
I think you should create the conversation object here and then send that as an argument to `CreateConversationPresenter.createCoversation` method which will add the object to Firebase.